pkg/cleaner/finalizer_cleaner.go

```diff
@@ -334,10 +334,14 @@ func checkFinalizerIsRemoved(cr FinalizerGetter, err error) (bool, error) {
 }
 
 func removeServiceCatalogFinalizer(finalizersList []string) []string {
+	const k8sIncubatorFinalizer = "kubernetes-incubator/service-catalog"
```

Comment:
```suggestion
	const k8sIncubatorFinalizer = "kubernetes-sigs/service-catalog"
```

Reply:
don't we have that const defined somewhere else already?

Reply:
we need to remove old and new finalizer, I'll add a comment